seed/challenges/01-front-end-development-certification/basic-ziplines.json

```diff
@@ -23,14 +23,14 @@
         [
           "http://i.imgur.com/p2TpOQd.jpg",
           "A cute dog jumping over a hurdle and winking and pointing his paw at you.",
-          "When you get stuck, just use the Read-Search-Ask methodology.<br>Don't worry - you've got this.",
+          "When you get stuck, just use the Read-Search-Ask methodology. Don't worry - you've got this.",
           ""
         ],
         [
-          "http://i.imgur.com/6WLULsC.gif",
+          "http://i.imgur.com/G1saeDt.gif",
           "A gif showing how to create a Codepen account.",
-          "For our front end project challenges, we'll use a popular browser-based code editor called CodePen. Open CodePen and click \"Sign up\" in the upper right hand corner, then scroll down to the free plan and click \"Sign up\" again. Click the \"Use info from GitHub button\", then add your email address and create a password. Click the \"Sign up\" button. Then in the upper right hand corner, click \"New pen\".",
-          "http://codepen.io"
+          "For our front end project challenges, we'll use a popular browser-based code editor called CodePen. Click the \"Open link in new tab\" button below to open CodePen's signup page. Fill out the form and click \"Sign up\". <br><div class=\"small\">Note: If you already have a CodePen account, you can skip this step by clicking \"Open link in new tab\", closing the new tab that opens, then clicking \"go to my next step\". We removed our \"skip step\" button because many people would just click it repeatedly without doing these important steps.</div>",
+          "http://codepen.io/signup/free"
         ],
```

Comment:
The alternative would be for everyone to have to go through an additional step (codepen.io/signup and then choose free, which is practically hidden on the page). I've chosen to bypass this. I suspect most people who already have CodePen accounts will skip this step anyway.